portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-details.js

```diff
@@ -1,25 +1,26 @@
 const { format, parseISO } = require('date-fns');
+const { getFormattedReportPeriod } = require('../../../helpers');
 
 /**
  * @typedef {Object} ReportAndUserDetails
  * @property {string} uploadedByFullName - The uploaded by users full name with format '{firstname} {surname}'
  * @property {string} formattedDateAndTimeUploaded - The date uploaded formatted as 'd MMMM yyyy at h:mmaaa'
- * @property {string} lastUploadedReportPeriod - The report period of the report formatted as 'MMMM yyyy'
+ * @property {string} lastUploadedReportPeriod - The report period of the report formatted as described in `getFormattedReportPeriod.ts`
```

Comment:
nit: Can link here using
```
{@link getFormattedReportPeriod}
```

Reply:
I tried that earlier and it didn't work - maybe my VSCode was just playing up!

Reply:
Tried it and it does work 👍